keyboards/acheron/shark/config.h

```diff
@@ -0,0 +1,259 @@
+/*
+Copyright 2019 Alvaro Volpato and Steve
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    AcheronProject
+#define PRODUCT         SharkPCB
+#define DESCRIPTION     Ortho 40 PCB
+
+/* key matrix size */
+#define MATRIX_ROWS 4
+#define MATRIX_COLS 12
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+*/
+#define MATRIX_ROW_PINS {B4, A15, B10, B2}
+#define MATRIX_COL_PINS {B1, B12, A1, A7, A5, A4, A3, A2, A0, C15, C14, C13}
+//{C13, C14, C15, A0,A2,A3,A4,A5,A6, A1,B12,B1}
+/* Unused pins, in the default layout, are:
+EncA (B6) because it is not used in the default PCB
+All Extra pins (A8, B15, B14, B13, B3, B5, B8, B9) , for the same reason;
+B0, which is unconnected on the PCB
+*/
+//#define UNUSED_PINS { B0, B6, B13, B14, B15, B8, B9, B5, B3 }
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+#define BACKLIGHT_PIN A6
```

Comment:
If you want backlight to actually work, you will need some of the other defines <https://docs.qmk.fm/#/feature_backlight?id=arm-configuration>, docs have pwm as TIM3_CH1 (which is enabled by default in `quantum/stm32/mcuconf.h`) so would be something like....

```suggestion
#define BACKLIGHT_PIN A6
#define BACKLIGHT_PWM_DRIVER PWMD3
#define BACKLIGHT_PWM_CHANNEL 1
```

*Note:* this would currently conflict with the ARM ws2812 support that is in the PR queue.

Reply:
If the conflict is due to the SPI/PWM conflit on A6, I did wire the RGB data-in to A6 (SPI1 MOSI) and the LED PWM to B0 (TIM3 CH3)

For now it would suffice to disable RGB and just make LEDs function if possible.

Reply:
The conflict is potentially due to the `arm_rgb` branches code assuming ownership of all channels on 2 PWM times, one of which is PWM3. However as you have used an SPI pin, that should be fine if we can merge the SPI rgb PR.

Reply:
New PWM pin assignment implemented in commit [c61af7](https://github.com/qmk/qmk_firmware/pull/7090/commits/c61af7c2f9924b75be45a601a0f5cb819905b512)